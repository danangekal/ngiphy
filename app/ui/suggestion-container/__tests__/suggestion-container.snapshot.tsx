import { render } from '@testing-library/react';

import SuggestionContainer from '../index';

const mockData = ['Dogs', 'Cats'];
const mockAction = jest.fn();
jest.mock('next/navigation', () => ({
  useRouter() {
    return {
      push: mockAction,
    };
  },
}));
const Component = <SuggestionContainer data={mockData} />;

it('SuggestionContainer unchanged', () => {
  const { container } = render(Component);
  expect(container).toMatchSnapshot();
});
