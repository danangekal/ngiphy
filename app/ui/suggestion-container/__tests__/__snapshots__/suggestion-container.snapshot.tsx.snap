// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuggestionContainer unchanged 1`] = `
<div>
  <div
    class="flex flex-col w-full items-center justify-between"
  >
    <div
      class="flex flex-row space-x-1 py-4 text-sm text-blue-500 "
    >
      <svg
        class="w-4 h-4"
        fill="none"
        stroke="currentColor"
        stroke-width="1.5"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M2.25 18 9 11.25l4.306 4.306a11.95 11.95 0 0 1 5.814-5.518l2.74-1.22m0 0-5.94-2.281m5.94 2.28-2.28 5.941"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
      <p>
        Trending Search
      </p>
    </div>
    <div
      class="grid grid-cols-1 sm:grid-cols-3"
    >
      <button
        class="border border-blue-500 text-blue-500 text-xs truncate me-2 px-2.5 py-0.5 my-0.5 rounded-full
              
            "
      >
        Dogs
      </button>
      <button
        class="border border-blue-500 text-blue-500 text-xs truncate me-2 px-2.5 py-0.5 my-0.5 rounded-full
              
            "
      >
        Cats
      </button>
    </div>
  </div>
</div>
`;
